### ISSUE TITLE
Fix Request: pmd7 AvoidMutableStaticFields false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#405]
```
import java.util.Arrays;
import java.util.List;

public class Issue {
    private final List<String> keys = Arrays.asList("key1", "key2"); // <--- false positive
}
```
